stripper/ze_FFXII_Westersand_v7_z9.cfg

```diff
@@ -9,4 +9,52 @@ modify:
 	{
 		"UseLandmarkAngles" "1"
 	}
-}
\ No newline at end of file
+}
+
+;Lock Holy and Earth Staff on all endings to prevent trolling except for Normal
+add:
+	"classname" "logic_relay"
+	"targetname" "Staff_Lock"
+	"origin" "-3156 -3083 2133"
+	"StartDisabled" "0"
+	"Spawnflags" "0"
+	"OnTrigger" "Staff_Earth_Button,Kill,0,1"
+	"OnTrigger" "Staff_Holy_Button,Kill,0,1"
+	"OnTrigger" "cmd,Command,say ** Locked Holy and Earth **,0,1"
+}
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"targetname" "ZE_Hard_Ending"
+	}
+	insert:
+	{
+		"OnStartTouch" "Staff_Lock,Enable,46,1"
+	}
+}
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"targetname" "ZE_Extreme_Ending"
+	}
+	insert:
+	{
+		"OnStartTouch" "Staff_Lock,Enable,56,1"
+	}
+}
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "Airship_Ending_Relay"
+	}
+	insert:
+	{
+		"OnTrigger" "Staff_Lock,Enable,0.5,1"
+	}
+}
```

Comment:
I don't think it's necessary to enable this on hard too. More fucking around lasers than actual tryhard lasers.

Additionally, you need to "Trigger" this relay to actually fire the outputs, it's already enabled by default.